src/tools/prism/language-definition.ts

```diff
@@ -16,4 +16,16 @@ export const languageDefinition = {
     greedy: true,
   },
   [OpenFgaDslThemeTokenType.DIRECTLY_ASSIGNABLE]: /\[.*]|self/,
+  [OpenFgaDslThemeTokenType.CONDITION]: {
+    pattern: /(\bcondition\s+)\w+/i,
+    lookbehind: true,
+    greedy: true,
+  },
+  "condition-params": {
+    pattern: /\(.*\)\s*{/,
+    inside: {
```

Comment:
Only thing I can see here that you might want to capture is the `,` separator as punctuation. Since its not already part of the language definition, and the params render as expected, probably not worth it.